linera-execution/src/lib.rs

```diff
@@ -414,6 +416,9 @@ pub trait ServiceRuntime: BaseRuntime {
         queried_id: UserApplicationId,
         argument: Vec<u8>,
     ) -> Result<Vec<u8>, ExecutionError>;
+
+    /// Get a blob of bytes from an arbitrary URL.
```

Comment:
Nit: conjugation in the documentation

```suggestion
    /// Gets a blob of bytes from an arbitrary URL.
```